inference-engine/samples/speech_sample/speech_sample.hpp

```diff
@@ -27,6 +27,23 @@ static const char target_device_message[] = "Optional. Specify a target device t
                                             "below. "
                                             "The sample will look for a suitable plugin for device specified.";
 
+/// @brief message for execution target
+static const char execution_target_message[] = "Optional (aka, GNA_EXEC_TARGET). Overrides GNA execution target generation. "
+                                               "Should be one of GNA_TARGET_2_0, GNA_TARGET_3_0 or empty. "
+                                               "When empty default value assumed: "
```

Comment:
I'd consider changing the help text a bit by omitting the "When empty" phrase, but leaving "default value" and its description. I'd rephrase the description also, currently it may imply that user should choose an option based on the presence of HW.

Reply:
done